common/model-views.cpp

```diff
@@ -2917,6 +2921,102 @@ namespace rs2
                         }
                     }
                 }
+
+                // Draw IR reflectivity on depth frame
+                if (!viewer.skip_option(RS2_OPTION_ENABLE_IR_REFLECTIVITY))
+                {
```

Comment:
Can we separate this block into a `draw_reflectivity` function or some such?

Reply:
And perhaps find a way to reuse for the IR frame, as well?

Reply:
Done